docs/api/storage/psa_protected_storage.md

```diff
@@ -0,0 +1,17 @@
+## PSA protected storage
+
+PSA protected storage APIs enable saving data to and retrieving data from PSA protected storage.
+
+Unlike [PSA internal storage](../apis/psa_internal_storage.html), PSA protected storage always runs in the Non-Secure Processing Environment (NSPE) and redirects calls to the KVStore static API.
+
+<span class="notes">**Note:** In general, we recommend using the [KVStore static API](../storage/KVStoreGlobalAPI.html) in the NSPE.</span>
+
+### PSA protected storage class reference
+
+[![View code](https://www.mbed.com/embed/?type=library)](../mbed-os-api-doxy/protected__storage_8h.html)
```

Comment:
I didn't think the Doxygen transclusions worked with the ../ links. Have you tried this in the preview? If you have and it does, I'll have to start using this.

Reply:
@AnotherButler, it seems to work :) Have a look: http://docs-previewer.herokuapp.com/mbed-os-5-docs/PSA-docs/apis/psa-protected-storage.html.